spec/controllers/forms_controller_spec.rb

```diff
@@ -139,6 +143,17 @@
             submit
             expect(response).to redirect_to thank_you_form_url(@form)
           end
+          it 'updates the current_user object' do
+            [:email, :first_name, :last_name].each do |attribute|
+              expect(@current_user.send(attribute))
```

Comment:
@Anbranin I think you might've been responding to something other than what @werebus wrote. These lines do seem like they're setting up the test, or more specifically confirming that the test setup worked properly, as opposed to asserting an expectation about the behavior of the website code.